beta/src/content/community/meetups.md

```diff
@@ -95,8 +95,8 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 * [Budapest](https://www.meetup.com/React-Budapest/)
 
 ## India {/*india*/}
-* [Bangalore](https://www.meetup.com/ReactJS-Bangalore/)
-* [Bangalore](https://www.meetup.com/React-Native-Bangalore-Meetup)
+* [Bengaluru](https://www.meetup.com/ReactJS-Bangalore/)
```

Comment:
Other occurrences have Bangalore in their name. Lets have the same here too? https://reactjs.org/community/conferences.html#react-nexus-2023

Reply:
Well. The city's name was changed long ago. If we keep writing the old name everytime and don't even try to do something about it. It's a lose lose.